plugins/org.polarsys.capella.vp.requirements.doc/html/3_userguide/35_propertiesview.html

```diff
@@ -57,7 +57,7 @@ <h2 id="Relations_between_elements_stored_in_different_layers">Relations between
 		</ol>
 		<p>
 			<i>Some limitations should be noted for Libraries (see the suitable part of 
-				<a href="/wiki/../help/topic/org.polarsys.capella.vp.requirements.doc/html/4_knownissues/4.1_limitations.html" title="../help/topic/org.polarsys.capella.vp.requirements.doc/html/4_knownissues/4.1_limitations.html">Limitations</a> section).
```

Comment:
need add in associated 35_properties.mediawiki the online documentation (span with class="github-link")

Reply:
Done